### ISSUE TITLE
The book needs a title

### ISSUE THREAD

**Issue by semioticrobotic** [open-organization-ambassadors/open-org-it-culture#20]
So our book needs a name.

Both our research and our interviews with source matter experts consistently revealed that terms like "culture," "IT culture," "culture change," and "IT culture change" resonate with would-be readers. As we've been noodling with title and subtitle ideas, then, we've been toying with variations on this theme.

As we wrestle with the important task of titling this volume, we're seeking community input. Below is a list of title and subtitle ideas; check them out, comment on your favorite, pair a few together to see how they sound, and offer your own suggestions. We'd love to have your feedback before we make a final decision.

## Titles

IT Starts with Culture  
IT's Open  
Open IT  
Reformatting IT  
The Open Organization IT Handbook  
The Open Organization Guide to IT Culture Change  
The Open Organization Guide to IT Innovation

## Subtitles

An open leader's guide to the future of IT  
Enhancing IT performance the open way  
The open organization guide to IT culture change  
The open organization guide to IT innovation  
Creating a culture of success in your IT shop  
Creating an open organization in your IT department  
Catalyzing cultures of continuous improvement

**Comment by janapetersen** [comment]
For titles: (1) The Open Organization Guide to IT Culture Change and (2) The Open Organization Guide to IT Innovation
For subtitles: I like (1) Enhancing IT performance the open way (2) creating a culture of success in your IT shop and (3) catalyzing (IT) cultures of continuous improvement

**Comment by semioticrobotic** [comment]
Thanks for weighing in, @janapetersen! I do agree that:

**The Open Organization Guide to IT Innovation: Catalyzing IT cultures of continuous improvement**

has a particularly nice ring to it.

**Comment by jhibbets** [comment]
@semioticrobotic I might flip it around: 

Catalyzing cultures of continuous improvement
The Open Organization Guide to IT Innovation

Part of The Open Organization community book series

**Comment by semioticrobotic** [comment]
Even better, @jhibbets!

I'm also partial to the subtitle "An open leader's guide to the future of IT." Just haven't yet found a productive or resonant coupling for that one.

**Comment by jhibbets** [comment]
We should conduct a poll from the top 4-5 ideas.

**Comment by rerb** [comment]
I vote (do I get a vote?) for "The Open Organization IT Handbook".  Simple, clear, maintains link to Open Organization idea.

**Comment by semioticrobotic** [comment]
These are all great suggestions, folks. Thanks. We're taking everything under consideration.

**Comment by semioticrobotic** [comment]
Some suggestions we've received via email:

THE OPEN ORGANIZATION  
Guide to IT Culture Change  
Creating a culture of success in your IT department

THE OPEN ORGANIZATION  
Guide to IT Culture Change  
Inspire your IT team and move faster

THE OPEN ORGANIZATION  
Guide to IT Culture Change  
Cultivating transparency, accountability, and failure to move faster

**Comment by arob98** [comment]
What terms or words did SMEs not respond favorably too? Sometimes seeing what is on the 'do not use' list can be helpful.

**Comment by semioticrobotic** [comment]
That's a great question, @arob98, and definitely something we specifically sought when asking for feedback. In the end, we decided to avoid:

* "agile IT" (it's too generic and passé at this point; plus, there's ambiguity inherent to the A/agile method/descriptor relationship)
* "DevOps" (plenty of contributors use this term to describe what they're doing, and some even deploy it specifically in their chapters; however, we consider the DevOps movement a particular _instance_ or _formulation_ of "OpenOrg" thinking more broadly (it derives its principles from the same set of values, in other words), and we didn't want to pigeonhole the book by putting the DevOps moniker in the title)
* "innovation" (this was a tricky one, because in many ways the word does cut to the heart of what the essays in the book espouse, and summarizes them well; however, on its own, it's just too generic, trendy, and vapid to really accomplish anything, so we prefer more concrete terms, as in the "transparency, accountability, and failure" example above)

**Comment by semioticrobotic** [comment]
Via email, the excellent @janapetersen recommends something that perhaps sounds like:

The Open Organization 
Guide to IT Culture Change
* (how to) Practice/ing transparency, accountability, and collaboration for continuous innovation
* (how to) Cultivate/ing the principles of open for continuous innovation
* (how to) Embed/ding principles of open for continuous innovation

**Comment by chris-short** [comment]
I like the "stop words" above (and I'm DevOps'ing atm). How about this?

The Open Organization
Guide to IT Culture Change
The Art of Creating Cultures and Moving Faster

**Comment by semioticrobotic** [comment]
Okay, folks. After much deliberation and stellar feedback from the community (both here and via email), we're proud to present the final four finalists for consideration. We'll make this official in the next day or two, so please add your final feedback soon, should you have anything additional to say.

And lest I forget: _Thank you_ for your help and guidance!

## The Final Four

THE OPEN ORGANIZATION
Guide to IT Culture Change
Creating a culture of success in your IT department

THE OPEN ORGANIZATION
Guide to IT Culture Change
Cultivating transparency, accountability, and collaboration in the IT department

THE OPEN ORGANIZATION
IT Culture Handbook
Open principles and practices for a more innovative IT department

THE OPEN ORGANIZATION
IT Culture Handbook
Enhancing transparency, accountability, and collaboration in the IT
organization

**Comment by semioticrobotic** [comment]
Well, folks, we did it.

Our final call for feedback provided the last bit of guidance we needed to polish the book's name. The fifth book in the _Open Organization_ book series will be titled:

THE OPEN ORGANIZATION  
Guide to IT Culture Change  
Open principles and practices for a more innovative IT department

A few notes on how we arrived at this decision:

* **Guide to IT Culture Change**: This title retains the familiar feel of previous community-produced companions by conveying a sense of utility; it's a "guide," a useful and practical resource for accomplishing something. Also in contention was the term "handbook," but community members ultimately felt "guide" sounded less intimidating and more accessible. What's more, this is a guide to _culture change_, a phrase that recurred across so many of our informational interviews and research calls with authors and source matter experts. Consensus was clear here: The biggest problems IT organizations are facing today aren't technical—they're cultural. That's why a book about something like "openness" is not only relevant but necessary today, and the title must absolutely stress this.

* **Open principles and practices for a more innovative IT department**: In the end, we managed to recuperate a term we initially sought to avoid because of its ambiguity: "innovation." While phrases like "culture of innovation" and "generating innovation" rang too hollow to really express the aim of the book, we think it works rather well in this particular instance, as it gestures toward the _kind_ of IT department our interviewees and authors described as imperative in the face of digital transformation. The evocative phrase "principles and practices" comes to us from contributor Justin Holmes, who offered it as a way of framing the book's twin focus on "cultures" (in Part 1) and "skills" (in Part 2)—on the abstract and the concrete. In the end, this subtitle/tagline seemed more descriptive and effective than the slightly more nebulous "Creating a culture of success in your IT department," and it allows us to "double down" on the notion of "open" as an essential component of IT culture change today (it also allows us to avoid some redundancy with the word "culture" appearing twice).

When we produced previous books, we simply compiled and edited the material, brainstormed a few relevant titles, shopped those to our friends and colleagues, performed a bit of SEO work, and settled on something. The process was straightforward and quick. But what's true of the open source approach to just about anything is true here, as well: Opening up decision-making ability to a community of interested and engaged participants will yield better results—even if those results take longer to land. Now, land they have—and we're much better off than we'd been had we gone it alone.

Thanks, all.
